src/db/connection.js

```diff
@@ -27,4 +27,12 @@ const closeDatabase = async (drop = false) => {
   await mongoose.connection.close();
 };
 
-module.exports = { connectToMongo, closeDatabase };
+const clearDatabase = async () => {
+  const { collections } = mongoose.connection;
+  /* eslint-disable no-restricted-syntax, guard-for-in, no-await-in-loop */
+  for (const key in collections) {
+    await collections[key].deleteMany();
+  }
+};
+
+module.exports = { connectToMongo, closeDatabase, clearDatabase };
```

Comment:
On line 15 of this file where you log `Database connected`, can you add the DB URL to that log statement?
This will help us verify that the correct DB connection is happening on normal server run and test run.